tutor/views.py

```diff
@@ -27,7 +30,12 @@ def tutor(request):
         HttpResponse: A generated http response object to the request depending on whether or not
                       the user is authenticated.
     """
-    if request.user.is_authenticated:
-        return render(request, "tutor/tutor.html")
-    else:
-        return render(request, "accounts/login.html")
+    lesson1 = Lesson.objects.get(lesson_name='testLesson')
```

Comment:
You should always check if there is a lesson that can be loaded. Get will fail if it can't locate a lesson with that name... We may need to figure out a way to quickly load the lessons into the database for testing purposes.

Reply:
Perhaps comment out lines 33 and 34 and adjust 35 accordingly...